drivers/sensor/stm32_temp/stm32_temp.c

```diff
@@ -67,10 +66,12 @@ static int stm32_temp_channel_get(const struct device *dev,
 		return -ENOTSUP;
 	}
 
+	int32_t mv = data->raw * cfg->tsv_mv / 0x0FFF; /* Sensor value in millivolts */
```

Comment:
I wonder if this shouldn't be float here ?

Reply:
I've checked it, and there is no difference with original implementation:

original code:
`data->mv = data->sample_buffer * cfg->tsv_mv / 0x0FFF;`
with `data->sample_buffer` being `int16_t` and `data->mv` being `int32_t`
I think you can loose here 1mv in precision, which is with a 12bit adc with Vref on 3V like 1 LSB.

I rather don't touch this since I cannot test the difference.